pkg/scheduler/api/job_info.go

```diff
@@ -51,6 +53,10 @@ type TaskInfo struct {
 	VolumeReady bool
 
 	Pod *v1.Pod
+
+	// Indicates whether or not the task is a backfill task. Either persist
+	// across sessions, or add this flag in pod annotation.
+	IsBackfill bool
```

Comment:
prefer to use slice, e.g. []Condition, Backfill is one of condition in running status.

Reply:
fixed